src/util/api/index.js

```diff
@@ -61,9 +62,20 @@ export const getUpdateReaderBookmarkAPI = url =>
 export const getFetchSuggestArticlesApi = () => `/api/reader/works/suggest`;
 export const getFetchFollowedSuperLikeApi = () =>
   `/api/reader/superlike/followed`;
-export const getOAuthRegisterAPI = (from = '', referrer = '') =>
-  `/api/users/register?from=${from}&referrer=${encodeURIComponent(referrer)}`;
-export const getOAuthLoginAPI = () => '/api/users/login';
+export const getOAuthRegisterAPI = (
+  language = 'zh',
+  from = '',
+  referrer = ''
+) =>
```

Comment:
Would using named param better? These 3 params are all optional, and doesn't have a logical relationship that one must be placed earlier than the others.

Reply:
Updated